basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,186 @@
-const helpers = require('./project-1');
+const helpers = require("./project-1");
 
 // start testing!
+/*
+- Use `yarn` to install the necessary dependencies 
+from the `package.json` file.
+- type `yarn test` to start the test runner.
+- open the `basic-javascript` folder and use `
+project-1.spec.js` to write tests for the functions 
+inside `project-1.js`.
+- use `project-2.spec.js` to write tests for the 
+functions inside `project-2.js`.
+
+As you write your tests, bugs in the code may be 
+revealed, feel free to correct and _refactor_ the 
+source code to satisfy your tests.
+*/
+describe("multipliesByTen", () => {
+  test("multiplies properly", () => {
+    const answer = helpers.multiplyByTen(2, 10);
```

Comment:
You should test for all other cases, what will happen if You pass same numbers? What will happen if You pass null value or string?